bootstrap/src/lib/util.ts

```diff
@@ -22,6 +22,15 @@ export const toObjectWithNumbers = (obj: any) => {
   return Object.fromEntries(Object.entries(obj).map(([k, v]) => [k, toNumber(v)]))
 }
 
+// pick a random string after splitting with the delimiter ("a&b&c" "&" -> choice(["a","b","c"]))
+export const pickRandomFromString = (str: string, delimiter: string) => {
```

Comment:
We can define the default delimiter here:

```suggestion
export const pickRandomFromString = (str: string, delimiter = ',') => {
```

Reply:
👍 included `,` as the default delimiter